tests/span_tests.cpp

```diff
@@ -483,6 +483,66 @@ SUITE(span_tests)
 #endif
     }
 
+    TEST(from_unique_pointer_construction)
+    {
+        {
+            auto arr = new int[4];
+            auto ptr = std::unique_ptr<int>(arr);
+
+            for (auto i = 0; i < 4; i++)
+                arr[i] = i + 1;
+
+            {
+                span<int> s{arr, 4};
```

Comment:
Is `arr` here supposed to be `ptr`? I assumed you were trying to test construction of the `span` from the `unique_ptr`.